packages/sysutils/ntfs-3g_ntfsprogs/package.mk

```diff
@@ -1,12 +1,13 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
```

Comment:
Looking at the package history this should be `2016-present` .. sorry.

Reply:
All fixed. all good.